abstar/utils/output.py

```diff
@@ -240,7 +241,7 @@ def _build_json_output(self, raw=False):
                         'score': self.antibody.j.score,
                         'assigner_score': self.antibody.j.assigner_score,
                         'others': [{'full': o.full,
-                                    'score': o.assigner_score}
+                                    'assigner_score': o.assigner_score}
```

Comment:
Renaming this since for every other reference in the codebase, `o.assigner_score` have been assigned to the `assigner_score` key.